### ISSUE TITLE
Missing migration

### ISSUE THREAD

**Issue by luxcem** [callowayproject/django-categories#124]
It seems that the package is missing a migration, whenever I run `python manage.py makemigrations`, django creates a new migration file that contains the following code : 

```python
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

from django.db import migrations, models
import django.db.models.manager

class Migration(migrations.Migration):

    dependencies = [
        ('categories', '0001_initial'),
    ]

    operations = [
        migrations.AlterModelManagers(
            name='category',
            managers=[
                ('tree', django.db.models.manager.Manager()),
            ],
        ),
        migrations.AlterField(
            model_name='category',
            name='thumbnail',
            field=models.FileField(blank=True, null=True, storage=[...], upload_to='uploads/categories/thumbnails'), # storage is set to my custom Storage Class
        ),
        migrations.AlterField(
            model_name='categoryrelation',
            name='relation_type',
            field=models.CharField(blank=True, help_text="A generic text field to tag a relation, like 'leadphoto'.", max_length=200, null=True, verbose_name='relation type'),
        ),
    ]
```

**Comment by epicserve** [comment]
@luxcem, this has been on my todo list for awhile. I try to get this done soonish.

**Comment by epicserve** [comment]
I released version 1.5.1, which should take care of this issue. On a project where I use django-categories, I had to do the following because I had already made migrations for django-categories.

1. Remove any category migrations that were auto created that are greater than 0001 from the django_migrations table.
2. You will also need to remove the migration files greater than 0001 from site-packages/categories/migrations/.
3. Then upgrade to 1.5.1 and run migrations for your project.
